### ISSUE TITLE
[BitMap] Bitset not support get(Long ...x1)

### ISSUE THREAD

**Issue by huazaihuola** [redisson/redisson#5216]
[BitMap] Bitset not support get(Long ...x1)
I find no this method ， so， we need to use "for or while " to get data， can we add  this method get(Long ...x) to the project.

**Comment by mrniko** [comment]
Do you mean to implement `List<Long> getLong(long... offsets)` method?

**Comment by huazaihuola** [comment]
I mean this 
`boolean get(long bitIndex);` boolean[] get(long ...bitIndex)
@mrniko

**Comment by mrniko** [comment]
as workaround you can use RBatch object

**Comment by huazaihuola** [comment]
I don't think this is a good solution. In the following scenario, I need to check if a list of user IDs (List<Long>) is in my blacklist.

`Set<Long> userIds = new HashSet<>(contentUserIds.size());
        RBatch batch = redissonClient.createBatch();
        for (Long contentUserId : contentUserIds) {
            batch.getBitSet(USER_FOLLOW + userId).getAsync(contentUserId);
        }

        BatchResult<?> execute = batch.execute();

        for (int i = 0; i < execute.getResponses().size(); i++) {
            Boolean value = (Boolean) execute.getResponses().get(i);
            Long contentUserId = contentUserIds.get(i);
            if(value!=null && value) {
                userIds.add(contentUserId);
            }
        }`

**Comment by mrniko** [comment]
`boolean[] get(long ...bitIndex)` method should be added

**Comment by mrniko** [comment]
Merged
